scripts/install.sh

```diff
@@ -569,4 +571,9 @@ function main() {
     report_install_success
 }
 
-main "$@"
+# tee logs into /var/log/kurl/install-<date>.log and stdout
+mkdir -p /var/log/kurl
+LOGFILE="/var/log/kurl/install-$(date +"%Y-%m-%dT%H-%M-%S").log"
+main "$@" 2>&1 | tee -a >(grep -vwE "(password)") $LOGFILE
```

Comment:
I could not get the `tee`d grep to work?

```sh
echo -e "Installing my awesome app\nYour temporary password is 'green'\nInstallation complete" | tee >(grep -vwE "(password)") my.log

...
[evans] $ cat my.log
Installing my awesome app
Your temporary password is 'green'
Installation complete
```

It works the other way though

```sh
[evans] $ echo -e "Installing my awesome app\nYour temporary password is 'green'\nInstallation complete" | grep -vwE "(password)"
Installing my awesome app
Installation complete
```

Reply:
Hi @banjoh, 

It is fixed now. Good catcher, thx. Following the test made:

- a) Get the latest script `curl https://kurl.sh/latest > install.sh`
- b) Run `chmod +x install.sh`
- c) Change the script with the same done in: https://github.com/replicatedhq/kURL/pull/3891/files

<img width="818" alt="Screenshot 2022-12-28 at 12 52 28" src="https://user-images.githubusercontent.com/7708031/209815264-9abd5c3c-d489-4af0-b27c-d6f7e6d64072.png">

- e) Add in the main func the same print that will be done by kots example:

<img width="826" alt="Screenshot 2022-12-28 at 12 19 38" src="https://user-images.githubusercontent.com/7708031/209811272-4bed8ba6-b5d4-454c-a38d-05102a08845f.png">

- e) Run the sudo bash ./install
- d) Check that the line `"Login with password (will not be shown again): senha"`  cannot be found in the logs:

<img width="989" alt="Screenshot 2022-12-28 at 12 54 04" src="https://user-images.githubusercontent.com/7708031/209815483-5c5dbe19-fac3-428c-b9be-c7b1e427666c.png">

Also, to ensure that the script would fail after these changes:

- a) Now, let's run without sudo
- b) See that it finished with the error code:

<img width="911" alt="Screenshot 2022-12-28 at 12 55 23" src="https://user-images.githubusercontent.com/7708031/209815630-9fd10033-0e54-4a16-be3c-98a57be0f7c4.png">